doc/tae.rst

```diff
@@ -22,6 +22,10 @@ provide slightly different interfaces. You can see `SVM-example
       <algo> <instance> <instance specific> <cutoff time> <runlength> <seed> <algorithm parameters>
       python algo.py 123 0 10 25 12345 -param1 value1 -param2 value2 [...]
 
+  The first two parameters after the ``algo.py`` hold the instance name on which the algorithm is evaluated and  extra information about the instance (rarely used).
+  The third parameter gives the cutoff time (maximal running time) of the target algorithm is allowed to run and the fourth the runlength (maximal number of steps).
```

Comment:
The third parameter is the cutoff time, which is the maximal time the target algorithm is allowed to run. The fourth parameter is the runlength, which is the maximal number of steps an algorithm is allowed to run.